### PR TITLE
Add README for running and postprocessing sphere tests

### PR DESCRIPTION
**Pull request by jiahe23**

### CODE REVIEW

**Comment by jiahe23** [comment]
bors try

File: examples/hybrid/sphere/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Running the simulation
+
+Here is a runscript template for setting up simulations on caltech central hpc.
+```
+#!/bin/bash
+
+#SBATCH --mem=32G
+#SBATCH --cpus-per-task=8
+#SBATCH --job-name=e
```
Comment (szy21) [inline examples/hybrid/sphere/README.md:11]:
```suggestion
#SBATCH --job-name=$YOUR_JOB_NAME
```

**Review by szy21 — APPROVED** [review]

File: examples/hybrid/sphere/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# A pipeline to run the sphere simulations on the cluster
```
Comment (valeriabarra) [inline examples/hybrid/sphere/README.md:1]:
```suggestion
# A pipeline to run the sphere simulations on Caltech's central cluster
```

File: examples/hybrid/sphere/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Running the simulation
+
+Here is a runscript template for setting up simulations on caltech central hpc.
```
Comment (valeriabarra) [inline examples/hybrid/sphere/README.md:5]: I wouldn't call this a "runscript"

Comment (jiahe23) [reply examples/hybrid/sphere/README.md:5]: @valeriabarra  how about sbatch script?

**Comment by jiahe23** [comment]
bors try

**Pull request merged**